# Generated for NaoQi method ALTextToSpeech.stopAll
# This method stops all the pending tasks.

---


